package.json

```diff
@@ -5,7 +5,9 @@
   "scripts": {
     "start": "docusaurus start",
     "build": "docusaurus build",
-    "lint": "remark -f -q -u validate-links \"docs/**/*.mdx\" && npm run prettier -- --check",
+    "lint": "npm run validate-links && npm run spellcheck && npm run prettier -- --check",
+    "spellcheck": "cspell \"{docs,src}/**\"",
```

Comment:
If you want to split these out can you instead use `lint:links`, `lint:spelling`?

Reply:
Totally, done.